app/controllers/data_services_controller.rb

```diff
@@ -7,6 +7,6 @@ def index
   end
 
   def show
-    @data_service = DataService.includes(:organisation).find(params[:id])
+    @data_service = DataService.includes(:organisation, :data_resource).find(params[:id])
```

Comment:
We're not using the organisation association anymore, so we can remove it from the includes. However instead we could add publisher and creators so that we preload them rather than loading them from inside the template. You may need to add a `through` association to the DataService model for that to work - have a go and let me know if you need help :)